.github/workflows/export-mpk.yml

```diff
@@ -0,0 +1,117 @@
+name: Export module package
+
+on:
+  # Run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Always use full git shas when using external actions. Otherwise, you're prone to hijacking. With official Github actions, that risk is not so high, but it's a good habit nonetheless.
https://michaelheap.com/ensure-github-actions-pinned-sha/